lib/benchee/formatters/console/run_time.ex

```diff
@@ -263,4 +264,8 @@ defmodule Benchee.Formatters.Console.RunTime do
     |> :io_lib.format([-label_width, name, @ips_width, ips_format, slower])
     |> to_string
   end
+
+  defp duration_output(duration, unit) do
```

Comment:
👍 This was a bad thing to share, and a bad name when it was shared. Thanks for making it better!